lib/tool_belt/repository.rb

```diff
@@ -0,0 +1,50 @@
+require 'git'
+
+module ToolBelt
+  class Repository
+
+    attr_reader :name, :branch, :url, :version_branch, :main, :options
+
+    def initialize(name, args = {}, options = {})
+      @name = name
+      @branch = args[:branch]
+      @url = args[:repo]
+      @version_branch = args[:version_branch]
+      @main = args[:main]
+      @options = options
+    end
+
+    def self.parse(config)
+      config.options.repos.map do |repo|
+        self.new(*repo, namespace: config.options.namespace)
+      end
+    end
+
+    def branch!
+      git.checkout('origin/master')
+      git.reset_hard('origin/master')
```

Comment:
This assumes every repository uses master. Foreman repositories in some cases use develop for their default branch.

Reply:
Do you think we could add a field in the config file?